templates/order.php

```diff
@@ -54,7 +54,7 @@
                 <!-- Order Status -->
                 <div class="bnomics-order-status-wrapper">
                   <span class="bnomics-order-status-title" ng-show="order.status == -1" ng-cloak ><?=__('To confirm your order, please send the exact amount of <strong>BTC</strong> to the given address', 'blockonomics-bitcoin-payments')?></span>
-                  <span class="warning bnomics-status-warning" ng-show="order.status == -3" ng-cloak><?=__('Payment Expired (Use the browser back button and try again)', 'blockonomics-bitcoin-payments')?></span>
+                  <span class="warning bnomics-status-warning" ng-show="order.status == -3" ng-cloak><?=__('<b>PAYMENT EXPIRED</b> <br /><br /><a href="/index.php/checkout?reset=1">Click here</a> to try again.<br /><br /><p>If you already paid, your order will be processed automatically. <br />You can safely close this window.</p>', 'blockonomics-bitcoin-payments')?></span>
```

Comment:
`/index.php/checkout?reset=1` is breaking websites.
![Screenshot from 2019-10-28 02-14-46](https://user-images.githubusercontent.com/39666372/67644543-19024580-f92b-11e9-9e20-7fa07457578b.png)

Reply:
There are display differences in the text CSS. 
![Different text formatting](https://user-images.githubusercontent.com/39666372/67644777-2ae4e800-f92d-11e9-9e77-e78833828601.png)

Reply:
Fixed as we use div now instead of <p>.

Reply:
https://github.com/blockonomics/woocommerce-plugin/pull/160#discussion_r339374488
fixing it using wss call to get timestamp. will be there on next commit.